package-lock.json

```diff
@@ -5131,14 +5143,12 @@
         "wrappy": {
           "version": "1.0.2",
           "bundled": true,
-          "dev": true,
-          "optional": true
+          "dev": true
         },
         "yallist": {
           "version": "3.0.3",
           "bundled": true,
-          "dev": true,
-          "optional": true
```

Comment:
Please upgrade to npm@>=6.6 so that you don't remove all the `optional` properties that are added by npm@>=6.6.

See related https://stackoverflow.com/questions/54411377/package-lock-json-file-package-with-optional-true